ClosePrompt/source.extension.vsixmanifest

```diff
@@ -28,9 +25,7 @@
                     Version="[4.5,)" />
     </Dependencies>
     <Prerequisites>
-        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor"
-                      Version="[15.0,16.0)"
-                      DisplayName="Visual Studio core editor" />
+        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[16.0.28315.86,)" DisplayName="Visual Studio core editor" />
```

Comment:
@phoffman81 thank you for the update! This 16.0.28315.86 seems to be odd - won't it break 2013? I would suggest to make the same change as above?

Reply:
Oddly enough, I thought I'd set it up that way, must have missed that one. I've corrected it and checked in the update.